devel/simple-web-server/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.0
+
+name	  	    simple-web-server
+version		    3.0.0-rc3
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen
```

Comment:
Add your obfuscated email address. Can `openmaintainer` be added too?

Reply:
done